backend/tests/conftest.py

```
@@ -0,0 +1,35 @@
+
+@pytest.fixture(scope="function")
+def db_session():
+    """Creates a new database session for each test."""
+    db = TestingSessionLocal()
+    try:
+        yield db
+    finally:
+        db.rollback()  
+        db.close()
+
+@pytest.fixture(scope="function")
```

Ruff (0.8.2)

`scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)